docs/content/guides/developer/sui-101/shared-owned.mdx

```diff
@@ -1,20 +1,171 @@
+For more information on the types of objects that Sui supports, see [Object Ownership](/concepts/object-ownership.mdx).
 
-Examine and weigh the tradeoffs between shared and owned objects on a case-by-case basis. To get an idea of the implications involved with shared versus owned objects, compare the following use cases, each implemented with shared and owned objects: 
+## Example: Escrow
 
-- Escrow: [shared](https://github.com/MystenLabs/sui/blob/main/sui_programmability/examples/defi/sources/shared_escrow.move), [owned](https://github.com/MystenLabs/sui/blob/main/sui_programmability/examples/defi/sources/escrow.move)
-- Auction: [shared](https://github.com/MystenLabs/sui/blob/main/sui_programmability/examples/nfts/sources/shared_auction.move), [owned](https://github.com/MystenLabs/sui/blob/main/sui_programmability/examples/nfts/sources/auction.move)
-- Tic Tac Toe: [shared](https://github.com/MystenLabs/sui/blob/main/sui_programmability/examples/games/sources/shared_tic_tac_toe.move), [owned](https://github.com/MystenLabs/sui/blob/main/sui_programmability/examples/games/sources/tic_tac_toe.move)
+The Escrow example demonstrates the trade-offs between shared objects and owned objects by implementing the same application in both styles. It implements a service that allows two addresses to perform a trustless swap of objects with each other (a "trade") with the service holding their objects in escrow.
+
+### `Locked<T>` and `Key`
+[Code Sample](https://github.com/MystenLabs/sui/blob/main/examples/move/escrow/sources/lock.move)
+
+Both implementations use a primitive for locking values, which offers the following interface:
+
+```rust
+module escrow::lock {
+    public fun lock<T: store>(obj: T, ctx: &mut TxContext): (Locked<T>, Key);
+    public fun unlock<T: store>(locked: Locked<T>, key: Key): T
+}
+```
+
+Any `T: store` can be locked, to get a `Locked<T>` and a corresponding `Key`, and conversely, the locked value and its corresponding key can be consumed to get back the wrapped object.
+
+The important property that this interface provides is that locked values cannot be modified except by unlocking them first (and later relocking them).  Because unlocking consumes the key, tampering with a locked value can be detected by remembering the ID of the key that it was locked with.  This prevents situations where one party in a swap changes the object they are offering to reduce its value.
+
+### Owned Objects
+
+[Code Sample](https://github.com/MystenLabs/sui/blob/main/examples/move/escrow/sources/owned.move)
+
+The protocol for swapping via escrow implemented using owned objects starts with both parties locking their respective objects. This will be used to prove that the object has not been tampered with after the swap has been agreed. If either party does not want to proceed at this stage, they simply unlock their object.
```

Comment:
```suggestion
The protocol for swapping via escrow implemented using owned objects starts with both parties locking their respective objects. This will be used to prove that the object has not been tampered with after the swap has been agreed. If either party does not want to proceed at this stage, they just unlock their object.
```

Reply:
Nit: Avoid "simple" as its extra frustrating if for some reason it's not simple for you as a user.